src/test/java/org/folio/rest/support/builders/RequestRequestBuilder.java

```diff
@@ -180,7 +191,8 @@ public RequestRequestBuilder withRequestType(String requestType) {
       this.cancellationAdditionalInformation,
       this.cancelledDate,
       this.position,
-      this.pickupServicePointId);
+      this.pickupServicePointId,
+      tags);
```

Comment:
To make this consistent, change `tags` to `this.tags`. This is the case for all the `withXXX()` methods in this file, except `withTags()`.